CHANGELOG.md

```diff
@@ -4,6 +4,43 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v4.0.0](https://github.com/voxpupuli/puppet-dhcp/tree/v4.0.0) (2020-02-24)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-dhcp/compare/v3.4.0...v4.0.0)
+
+**Breaking changes:**
+
+- drop Ubuntu 14.04 support [\#230](https://github.com/voxpupuli/puppet-dhcp/pull/230) ([bastelfreak](https://github.com/bastelfreak))
```

Comment:
I've updated this PR to get a more logical changelog. Please have a look and regen.

Reply:
fixed